apps/avatax/src/modules/avatax/calculate-taxes/avatax-calculate-taxes-payload-lines-transformer.ts

```diff
@@ -34,7 +112,7 @@ export class AvataxCalculateTaxesPayloadLinesTransformer {
         amount: taxBase.shippingPrice.amount,
         taxCode: config.shippingTaxCode,
         taxIncluded: taxBase.pricesEnteredWithTax,
-        discounted: isDiscounted,
+        discounted: hasEntireCheckoutDiscount,
```

Comment:
**question**: Is this the only case when a shipping can be discounted?

Reply:
Your comment made me rethink this PR slightly. I changed it so that Avatax will ask for voucher type in the event subscription so shipping line will be discounted when voucher is for shipping.